{
  "id": "GHSA-v3rg-qm46-xrg9",
  "summary": "Path traversal in flaskcode",
  "details": "An issue was discovered in the flaskcode package through 0.0.8 for Python. An unauthenticated directory traversal, exploitable with a POST request to a /update-resource-data/<file_path> URI (from views.py), allows attackers to write to arbitrary files.",
  "aliases": [
    "CVE-2023-52289"
  ],
  "modified": "2024-02-16T08:16:07.416021Z",
  "published": "2024-01-13T06:30:26Z",
  "database_specific": {
    "nvd_published_at": "2024-01-13T04:15:08Z",
    "cwe_ids": [
      "CWE-22"
    ],
    "severity": "HIGH",
    "github_reviewed": true,
    "github_reviewed_at": "2024-01-24T21:53:43Z"
  },
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2023-52289"
    },
    {
      "type": "WEB",
      "url": "https://gitlab.com/daniele_m/cve-list/-/blob/main/README.md"
    }
  ],
  "affected": [
    {
      "package": {
        "name": "flaskcode",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/flaskcode"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "last_affected": "0.0.8"
            }
          ]
        }
      ],
      "versions": [
        "0.0.2",
        "0.0.3",
        "0.0.4",
        "0.0.5",
        "0.0.6",
        "0.0.7",
        "0.0.8"
      ],
      "database_specific": {
        "source": "https://github.com/github/advisory-database/blob/main/advisories/github-reviewed/2024/01/GHSA-v3rg-qm46-xrg9/GHSA-v3rg-qm46-xrg9.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:H/A:N"
    }
  ]
}